doc/releases/release-notes-3.7.rst

```diff
@@ -389,30 +389,71 @@ Build system and Infrastructure
 
   * A ``socs`` folder for applications has been introduced that allows for Kconfig fragments and
     devicetree overlays that should apply to any board target using a particular SoC and board
-    qualifier.
+    qualifier (:github:`70418`). Support has also been added to sysbuild (:github:`71320`).
 
-  * :ref:`Board/SoC flashing configuration<flashing-soc-board-config>` settings have been added.
+  * :ref:`Board/SoC flashing configuration<flashing-soc-board-config>` settings have been added
+    (:github:`69748`).
 
   * Deprecated the global CSTD cmake property in favor of the :kconfig:option:`CONFIG_STD_C`
     choice to select the C Standard version. Additionally subsystems can select a minimum
     required C Standard version, with for example :kconfig:option:`CONFIG_REQUIRES_STD_C11`.
 
-  * Fixed issue with passing UTF-8 configs to applications using sysbuild.
+  * Fixed issue with passing UTF-8 configs to applications using sysbuild (:github:`74152`).
 
   * Fixed issue whereby domain file in sysbuild projects would be loaded and used with outdated
-    information if sysbuild configuration was changed, and ``west flash`` was ran directly after.
+    information if sysbuild configuration was changed, and ``west flash`` was ran directly after
+    (:github:`73864`).
 
   * Fixed issue with Zephyr modules not being listed in sysbuild if they did not have a Kconfig
-    file set.
+    file set (:github:`72070`).
 
-  * Add sysbuild ``SB_CONFIG_COMPILER_WARNINGS_AS_ERRORS`` Kconfig option to turn on
-    "warning as error" toolchain flags for all images, if set.
+  * Added sysbuild ``SB_CONFIG_COMPILER_WARNINGS_AS_ERRORS`` Kconfig option to turn on
+    "warning as error" toolchain flags for all images, if set (:github:`70217`).
 
   * Fixed issue whereby files used in a project (e.g. devicetree overlays or Kconfig fragments)
-    were not correctly watched and CMake would not reconfigure if they were changed.
+    were not correctly watched and CMake would not reconfigure if they were changed
+    (:github:`74655`).
 
   * Added flash support for Intel Hex files for the LinkServer runner.
 
+  * Added sysbuild ``sysbuild/CMakeLists.txt`` entry point and added support for
+    ``APPLICATION_CONFIG_DIR`` which allows for adjusting how sysbuild functions (:github:`72923`).
+
+  * Fixed issue with sysbuild not failing if ``APPLICATION_CONFIG_DIR`` was set to an invalid
```

Comment:
maybe we don't need to mention a fix for a feature we just added in this same release?

Reply:
Removed